core/src/main/java/org/apache/calcite/rel/rules/CoreRules.java

```diff
@@ -850,4 +850,16 @@ private CoreRules() {}
   @Experimental
   public static final DphypJoinReorderRule HYPER_GRAPH_OPTIMIZE =
       DphypJoinReorderRule.Config.DEFAULT.toRule();
+
+  /** Rule that expand disjuction in condition of a {@link Filter}.
```

Comment:
this "disjunction" is still misspelled
it should also be "expands disjunction in the condition of a filter"
same problems in the comment below